docker-run.sh

```diff
@@ -40,6 +40,28 @@ if [[ "${QUERY_CONFIG_ENABLED}" = "true" ]]; then
   sed -i "s|queryConfigEnabled: false|queryConfigEnabled: true|" $INDEX_FILE
 fi
 
+if [[ "${URL_SWAGGER2_GENERATOR}" ]]; then
+  if [[ "$URL_SWAGGER2_GENERATOR" != "null" ]]; then
+    sed -i "s|swagger2GeneratorUrl: 'https://generator.swagger.io/api/swagger.json'|swagger2GeneratorUrl: '${URL_SWAGGER2_GENERATOR}'|g" $INDEX_FILE
```

Comment:
To add the `swagger2GeneratorUrl` as option to `SwaggerEditorBundle` function, following shell code needs to be introduced:

```suggestion
    sed -i "s|SwaggerEditorBundle({|SwaggerEditorBundle({\n      swagger2GeneratorUrl: '${URL_SWAGGER2_GENERATOR}',|g" index.html
```

This independently injects new options to `SwaggerEditorBundle` function and should be used through out the changes in `docker-run.sh` that this PR introduced.

Reply:
You're right, see my answer: https://github.com/swagger-api/swagger-editor/pull/2910#discussion_r821852314